src/utils/types.js

```diff
@@ -23,9 +23,13 @@ export const types = {
   welcomeMessage: PropTypes.oneOfType([
     PropTypes.bool,
     PropTypes.array,
-    PropTypes.string
+    PropTypes.string,
+    PropTypes.func
   ]),
-  promptLabel: PropTypes.string,
-  errorText: PropTypes.string,
-  commands: PropTypes.object.isRequired
+  commands: PropTypes.object.isRequired,
+  promptLabel: PropTypes.func,
```

Comment:
A better idea: Accept a node for this, instead of breaking existing code that uses the string variant.